### ISSUE TITLE
Quest: "Investigate the Blue Recluse"

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5439]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Apart from the spawn number issue I posted in another thread, the Rift Spawns do not become "stunned" at low health and thus cannot be captured by the containment coffers which means the quest cannot be done. If you attempt to kill the Rift Spawn it will just run away at 0 hp and there text log says "Rift Spawn escapes into the void!"

**Expected behaviour**: Tell us what should happen instead
The Rift Spawns should enter a "stunned" state at low health to be captured by the containment coffers
 
**Steps to reproduce the problem**:

1. Use the Cantation of Manifestation to summon the Rift Spawns in the Blue Recluse
2. Attack the Rift Spawn and observe it's behaviour at low health and when it's health hits zero
3. 

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/32976781/39519210-673fa20a-4e38-11e8-8775-edfc0502833e.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate and fixed in https://github.com/dalaranwow/dalaran-wow/issues/5417
